browserslist

```diff
@@ -1,7 +0,0 @@
-# Browser support list - used by CSS autoprefixer
-last 2 Chrome versions
-last 2 Firefox versions
-last 2 Safari versions
-last 2 Edge versions
-last 2 iOS versions
-ie 11
```

Comment:
Do we still need autoprefixing for IE11?

Reply:
I think so? Also AFAIK some moz- prefixes are needed too